fiftyone/core/collections.py

```diff
@@ -9054,6 +9089,15 @@ def get_index_information(self):
             # Frame-level indexes
             fields_map = self._get_db_fields_map(frames=True, reverse=True)
             frame_info = self._dataset._frame_collection.index_information()
+
+            if include_size:
```

Comment:
FYI- added support for frame indexes on video datasets

Reply:
Aha!